src/runtime/constructorbinder.cs

```diff
@@ -89,7 +89,7 @@ internal object InvokeRaw(IntPtr inst, IntPtr args, IntPtr kw, MethodBase info)
                 // any extra args are intended for the subclass' __init__.
 
                 IntPtr eargs = Runtime.PyTuple_New(0);
-                binding = Bind(inst, eargs, kw);
+                binding = Bind(inst, eargs, IntPtr.Zero);
```

Comment:
Why is this change necessary?

Reply:
The old code was actually incorrect, but it was not exposed until I made the bugfix in `Bind`. We are looking for a zero argument constructor here, so no kwargs should be passed in. As the comment above says: "any extra args are intended for the subclass' __init__."